.changeset/strong-seahorses-scream.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-backend': minor
```

Comment:
This is just an addition, so while we are on 0.x, this should actually be `patch`.